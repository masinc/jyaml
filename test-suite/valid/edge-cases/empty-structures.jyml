# Empty structures
"empty_object": {}
"empty_array": []
"empty_string": ""