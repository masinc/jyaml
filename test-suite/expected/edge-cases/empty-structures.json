{"empty_object": {}, "empty_array": [], "empty_string": ""}